assets/js/translator.js

```diff
@@ -802,6 +936,19 @@ function translationNotAvailable() {
         .addClass('notAvailable text-danger');
 }
 
+function translationNotAvailableWebpage(data) {
+    if(!data) {
+        return;
+    }
+    translationNotAvailable();
+    var div = $('<div id="translatedWebpage" class="translatedWebpage notAvailable text-danger"></div>')
+        .text(getDynamicLocalization('Not_Available'));
+    $('#translatedWebpage').replaceWith(div[0]);
+    $('#translatedWebpage').append([$('<div></div>').text(' '),
```

Comment:
This is really strange spacing :) Can you do something like ([
   1st item,
   2nd item,
   3rd item
])